test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -2033,39 +2033,75 @@ public void Mesh2dCasulliRefinementOnPolygonThroughApi()
             }
         }
 
-        [Test]
-        public void Mesh2dComputeOrthogonalizationThroughApi()
-        {
-            // Setup
-            using (DisposableMesh2D mesh = CreateMesh2D(4, 4, 100, 200))
-            using (var api = new MeshKernelApi())
-            using (var polygon = new DisposableGeometryList())
-            using (var landBoundaries = new DisposableGeometryList())
-            {
-                var id = 0;
-                DisposableMesh2D mesh2D = null;
-                try
-                {
-                    id = api.AllocateState(0);
+        [Test]
+        public void Mesh2dComputeOrthogonalizationThroughApi()
+        {
+            // Setup
+            using (DisposableMesh2D mesh = CreateMesh2D(4, 4, 100, 200))
+            using (var api = new MeshKernelApi())
+            using (var polygon = new DisposableGeometryList())
+            using (var landBoundaries = new DisposableGeometryList())
+            {
+                var id = 0;
+                DisposableMesh2D mesh2D = null;
+                try
+                {
+                    id = api.AllocateState(0);
+
+                    Assert.AreEqual(0, api.Mesh2dSet(id, mesh));
+                    var orthogonalizationParametersList = OrthogonalizationParameters.CreateDefault();
+
+                    Assert.AreEqual(0, api.Mesh2dComputeOrthogonalization(id,
+                                                                          ProjectToLandBoundaryOptions.ToOriginalNetBoundary,
+                                                                          orthogonalizationParametersList,
+                                                                          polygon,
+                                                                          landBoundaries));
+                    Assert.AreEqual(0, api.Mesh2dGetData(id, out mesh2D));
+                    Assert.AreEqual(16, mesh2D.NumNodes);
+                }
+                finally
+                {
+                    api.DeallocateState(id);
+                    mesh2D?.Dispose();
+                }
+            }
+        }
+
+        [Test]
+        public void Mesh2dConnectMeshesThroughApi()
+        {
+            // Setup
+            using (DisposableMesh2D firstMesh = CreateMesh2D(3, 3, 1, 1))
+            using (DisposableMesh2D secondMesh = CreateMesh2D(6, 6, 0.5, 0.5, 3.0))
+            using (var api = new MeshKernelApi())
+            {
+                var id = 0;
+                DisposableMesh2D mesh2D = null;
+                try
+                {
+                    // Prepare
+                    double searchFraction = 0.1;
+                    id = api.AllocateState(0);
+                    Assert.AreEqual(0, api.Mesh2dSet(id, firstMesh));
```

Comment:
No need to assert on return value. This stuff is tested elsewhere and it makes reading the test harder.